test/AutoDiff/autodiff_diagnostics.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -emit-sil -verify %s
+// RUN: %target-swift-frontend -emit-sil -verify -Xllvm -differentiation-enable-control-flow %s
```

Comment:
Todo: revert the accidental RUN line change. `-Xllvm -differentiation-enable-control-flow` should go away soon.
Will commit after CI passes.